hicommonwealth/commonwealth
# Added staked community filter in explore page

## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/6692

## Description of Changes
- Updated filter UI in global explore page
- Added `Stake` filter to filter out stake-enabled communities.

## "How We Fixed It"
N/A

## Test Plan
Pre-req: must have a stake-enabled community

1. Go to global explore page
2. Verify new filters match [figma](https://www.figma.com/file/omZZrndm6Phbk1rOU4IKhS/Stake-V1-Fast-Follows?type=design&node-id=1-15142&mode=design&t=6kVbTbejAMJhpAmo-0)
3. Verify that clicking on the `Stake` filter, filters out stake enabled communities from the list.

## Deployment Plan
N/A

## Other Considerations
N/A
